ui/src/app/components/card.tsx

```diff
@@ -9,10 +9,22 @@ export const styles = reactStyles({
     border: '1px solid #d7d7d7',
     display: 'flex',
     flexDirection: 'column',
-    width: '12rem',
-    height: '9rem',
     margin: '0 1rem 1rem 0'
   }
 });
 
-export const Card = withStyle(styles.card)('div');
+export const styles = reactStyles({
+  workspaceCard: {
+    ...baseStyles.card,
+    minWidth: '300px', maxWidth: '300px',
```

Comment:
adding both max and min for width/height should fix any issues with card size changing on page resize